boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
This line is a serious problem. You are mixing a synchronous flow with an asynchronous one. Caling a callback in a synchronous manner can lead to unexpected behaviors because the flow of the program is not going on the expect way.